http-server/src/test/resources/roles.properties

```diff
@@ -0,0 +1,3 @@
+user=p1
+internal=p2
+admin=p3
```

Comment:
lets make the test case more elaborate. Add some fancy regexes here instead of just simple text